docs/source/documentation.rst

```diff
@@ -503,89 +481,27 @@ Jupyter notebooks
 
```

Comment:
I found an error on lines 414 and 425 while reviewing (unrelated to this PR).
On both line, the command should be `axondeepseg_morphometrics` instead of `axondeepseg`.

Reply:
Done in 289a55c